src/smt/unsat_core_manager.cpp

```diff
@@ -46,7 +46,7 @@ void UnsatCoreManager::getUnsatCore(std::shared_ptr<ProofNode> pfn,
   Trace("unsat-core") << "UCManager::getUnsatCore: free assumptions: "
                       << fassumps << "\n";
   const context::CDList<Node>& al = as.getAssertionList();
-  std::unordered_set<Node> coreSet;
+  std::set<Node> coreSet;
```

Comment:
Unordered set is faster, we should continue to use std::unordered_set here.

Reply:
Using `set` ensures the order is the same across different platforms.